Assets/Scripts/StandardSamples/UI/Storage/Player/UIPlayerDataStorageMenu.cs

```diff
@@ -71,21 +60,17 @@ public class UIPlayerDataStorageMenu : MonoBehaviour, ISampleSceneUI
 
         public Dropdown AddItemDropdown;
 
-        [Header("Controller")]
-        public GameObject UIFirstSelected;
-
         private string currentSelectedFile = string.Empty;
 
         private EOSPlayerDataStorageManager PlayerDataStorageManager;
         private PlayerDataInventory currentInventory = null;
 
-        private HashSet<string> fileNames;
-        private List<UIFileNameEntry> fileNameUIEntries;
+        private HashSet<string> fileNames = new();
+        private List<UIFileNameEntry> fileNameUIEntries = new();
 
-        private void Awake()
+        protected override void OnEnable()
```

Comment:
I'm unsure we'll run in to this scenario, but this behaviour is different. I believe `Awake` can only run once, but `OnEnable` will run every time it becomes enabled. If someone shows, then hides, then shows this, it will have added `fileNameUIEntries` twice with the same data.

Reply:
You are correct. Not certain how this changed to `OnEnable`

Reply:
Ficksed!